spec/collectors/statsd.rb

```diff
@@ -32,6 +32,20 @@ module Collectors
         it 'records a single result' do
           expect(collector.results.count).to eq 1
         end
+
+        it 'Converts test description to 8 character hash' do
+          small = collector.formatDesc('test')
+          expect(small.length).to eq(8)
+          expect(small).to eq('ccb19ba6')
+          big = collector.formatDesc('test more strings with spaces_and_underscores')
```

Comment:
At least `expect` calls for small and big descriptions should be in different test cases. I personnaly agree with rubocop's recomendation to have only one expectation for each test, so if you want to check both length and value, you may make a context and write two tests. But I also think, that in this case it's the overkill, because when your expectation on a value is truthy, its length can't fail the test. So I'd recommend to separate tests for small and big values and remove length checks.

If something goes wrong with only descriptions with spaces and underscores, you'll see it clear that short description is processed well

Reply:
Sounds good, I thought of the redundancy too but figured it might catch a strange case where just the hash changed while the length was still correct.